changelogs/unreleased/6672-abbas-gheydi-small.md

```diff
@@ -0,0 +1,2 @@
+## **Access Log Enhancement**
```

Comment:
the PR number should be used in the changelog filename: 6077

Since this is categorized as a `small` change, we can omit the title and just include a one line changelog as well